Makefile

```diff
@@ -72,7 +72,7 @@ DOCKER_ENV := \
 		-v $(DOCKER_CACHE)/gopath:/tmp/gopath \
 		-v $(DOCKER_CACHE)/gocache:/tmp/gocache \
 		-v $(CURDIR):/usr/src/antrea.io/theia \
-		golang:1.20
+		golang:1.21
```

Comment:
Lets update go-lint also , latest version we may use is  v1.54.2 for line 121

Reply:
Lets have it updated automatically, you can refer to https://github.com/antrea-io/antrea/commit/c286ba32ce3fa68e4583101b21b45b5c01fed022